src/platform/mhv/hooks/useMyHealthAccessGuard.jsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+
+/**
+ * Route guard hook that will redirect the user to the /my-health landing page if mhvAccountState is 'NONE'
+ */
+
+export const useMyHealthAccessGuard = () => {
```

Comment:
Perhaps change the name of this function. The name kind of implies to me that the return would be boolean.

Reply:
The "use" prefix is to indicate that this is a custom hook.